### ISSUE TITLE
Migrate master to main: https://github.com/pytorch/tutorials 

### ISSUE THREAD

**Issue by janeyx99** [pytorch/pytorch#71843]
Please look at the parent issue for instructions.(https://github.com/pytorch/pytorch/issues/62540)

**Comment by ajcost** [comment]
@janeyx99 Could I work on this?

**Comment by janeyx99** [comment]
@ajcost Yes--If you're willing to make the corresponding PR in pytorch/tutorials, I can review + implement the internal-facing changes.

**Comment by ajcost** [comment]
@janeyx99 yeah sure I'll take a look this week
